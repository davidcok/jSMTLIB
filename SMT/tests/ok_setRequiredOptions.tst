; Tests whether the required options are settable
(set-option :print-success true)
(get-option :print-success)
(set-option :regular-output-channel "stdout")
(get-option :regular-output-channel)
(set-option :diagnostic-output-channel "stderr")
(get-option :diagnostic-output-channel)
(set-option :diagnostic-output-channel "stdout")
(get-option :diagnostic-output-channel)
(set-option :diagnostic-output-channel "tempout")
(get-option :diagnostic-output-channel)
(set-option :print-success false)
(get-option :print-success)
