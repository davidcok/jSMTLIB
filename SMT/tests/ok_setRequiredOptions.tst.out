success
true
success
"stdout"
success
"stderr"
success
"stdout"
success
"tempout"
false
